mil_common/drivers/mil_passive_sonar/scripts/ping_publisher.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import contextlib
+import json
+import socket
+
+import rospy
+from geometry_msgs.msg import Point
+from mil_passive_sonar.msg import ProcessedPing
+from std_msgs.msg import Header
+
+
+def main():
+    # Define the server address and port
+    HOST = "127.0.0.1"
+    PORT = 2007  # The port used by the server
+
+    # Create a socket
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.connect((HOST, PORT))
+        print(f"Connected to {HOST}:{PORT}")
+        pub = rospy.Publisher("ping", ProcessedPing, queue_size=10)
```

Comment:
I would name the topic related to "hydrophones", like "/hydrophones/solved", or "/hydrophones/locations"